setup.cfg

```diff
@@ -0,0 +1,19 @@
+[flake8]
```

Comment:
It'd be good to have a configured coding-standard checker as a dependency, this was started in https://github.com/elifesciences/proofreader-python.

We have https://github.com/libero/php-coding-standard which is applied across all the PHP repos and saves having to try and replicate the same configuration everywhere. It does only do the one tool though, but is the one that needs the fine-grained control.

Reply:
Good idea. I would like to see how this project evolves and, once well established, move the coding standard checks to its own repo. A separate issue should be created for this though and prioritised accordingly.